lib/cusolver/dense.jl

```diff
@@ -895,6 +900,7 @@ for elty in (:Float32, :Float64, :ComplexF32, :ComplexF64)
         LinearAlgebra.LAPACK.potrs!(uplo::Char, A::StridedCuMatrix{$elty}, B::StridedCuVecOrMat{$elty}) = CUSOLVER.potrs!(uplo, A, B)
         LinearAlgebra.LAPACK.potri!(uplo::Char, A::StridedCuMatrix{$elty}) = CUSOLVER.potri!(uplo, A)
         LinearAlgebra.LAPACK.getrf!(A::StridedCuMatrix{$elty}) = CUSOLVER.getrf!(A)
+        LinearAlgebra.LAPACK.getrf!(A::StridedCuMatrix{$elty}, ipiv::CuVector{Cint}) = CUSOLVER.getrf!(A, ipiv)
```

Comment:
FWIW, I'm no fan of direct implementations of `LinearAlgebra.LAPACK` or `LinearAlgebra.BLAS` with CUSOLVER/CUBLAS functionality; the APIs are often not really compatible, so I think it's safer to override at the `LinearAlgebra` level.